snowpack/src/commands/dev.ts

```diff
@@ -426,7 +430,7 @@ export async function startServer(
       encoding: _encoding,
       importMap,
     }: LoadUrlOptions = {},
-  ): Promise<LoadResult> {
+  ): Promise<LoadResult | undefined> {
```

Comment:
This is also an important change: allow the dev server to request possibly-undefined URLs without throwing. They’ll simply return an empty response if nothing was built. That can then be handled at a higher-level from what’s calling this.